### DISCUSSION TITLE
子应用之间公共vendors怎么共享

### DISCUSSION THREAD

**Discussion by zhuzhuaicoding** [umijs/qiankun#1842] [Q&A]
子应用1里面依赖lodash，子应用2里面也依赖lodash，怎么让2个子应用公用一个lodash？不然的话，2个子应用都会把lodash打包进去，这是合理的吗？

**Comment by speoth** [comment]
多入口打包？因为用同一个编译时，lodash应该是被打进一个chunk里的

**Comment by humorHan** [comment]
全局挂载了，虽然不太好，但是子应用多了实在耗不起

**Comment by magicstf** [comment]
放在主应用不就好了

**Reply by zhuzhuaicoding** [reply]
放到主应用里，通过props传进去又不优雅
